public/fonts/fonts

```diff
@@ -0,0 +1 @@
+node_modules/@mozilla-protocol/core/protocol/fonts
```

Comment:
It looks like you accidentally created the symbolic link *inside* the fonts directory. I forgot to mention that you would want to *delete* the contents of the directory before copying.

Now that I think about it though, I suspect this approach isn't going to work -- since there are also some assets we want to keep in this directory locally (e.g. the glean logo). I think a better approach is to copy the files from protocol *into* this directory at build time, using something like [rollup-plugin-copy](https://www.npmjs.com/package/rollup-plugin-copy). Can you give that a try?